{"name": "test_login_failed_user_password_wrong[case3]", "status": "passed", "parameters": [{"name": "case", "value": "{'user': 'aaaaaa', 'password': '', 'check': '缺少用户名或密码'}"}], "start": 1665209571145, "stop": 1665209571703, "uuid": "90b0f491-fb8f-44ea-838e-c6a76e2537dd", "historyId": "8d1ba2ad9c9c202aa080e5d737ea80f9", "testCaseId": "253d57a3ffef1d7866e3686ffe2eb276", "fullName": "TestCases.test_login.TestLogin#test_login_failed_user_password_wrong", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init', 'back_to_login_page')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-E380NSU"}, {"name": "thread", "value": "18584-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}